provider/resources.go

```diff
@@ -63,28 +65,29 @@ func makeDataSource(mod string, res string) tokens.ModuleMember {
 // Provider returns additional overlaid schema and metadata associated with the provider..
 func Provider() tfbridge.ProviderInfo {
 	// Instantiate the Terraform provider
-	p := shimv2.NewProvider(shim.NewProvider())
+	//p := shimv2.NewProvider(shim.NewProvider())
 
 	// Create a Pulumi provider mapping
 	prov := tfbridge.ProviderInfo{
-		P:           p,
+		P:           pf.ShimProvider(shim.NewProvider()),
 		Name:        "cloudinit",
+		Version:     version.Version,
 		Description: "A Pulumi package for creating and managing cloudinit cloud resources.",
 		Keywords:    []string{"pulumi", "cloudinit"},
 		License:     "Apache-2.0",
 		Homepage:    "https://pulumi.io",
 		Repository:  "https://github.com/pulumi/pulumi-cloudinit",
 		GitHubOrg:   "hashicorp",
 		Config:      map[string]*tfbridge.SchemaInfo{},
-		Resources: map[string]*tfbridge.ResourceInfo{
-			"cloudinit_config": {
-				Tok: makeResource(mainMod, "Config"),
-				Docs: &tfbridge.DocInfo{
-					Markdown: []byte(" "),
-				},
-				DeprecationMessage: "This resource is deprecated.\nPlease use the getConfig data source instead.",
-			},
-		},
+		//Resources: map[string]*tfbridge.ResourceInfo{
```

Comment:
commented out code

Reply:
I think we can delete `Resources` and `DataSources` here.